doc/htmldoc/benchmark_results.rst

```diff
@@ -0,0 +1,106 @@
+.. _nest_benchmark_results:
+
+NEST performance benchmarks
+===========================
+
+
+NEST performance is continuously monitored and improved across various network sizes.
+Here we show benchmarking results for NEST version 3.8 on Jureca-DC.
```

Comment:
replace: "for this version of NEST on JURECA-DC [x]."
Jureca is discribed in the large scale instrument Journal of Juelich. Please include the reference in the usual citation format. Here is the paper., it also has a doi: https://jlsrf.org/index.php/lsf/article/view/182/pdf